examples/mongoose-express/README.md

```diff
@@ -0,0 +1,254 @@
+   │    └── 3-orders // `orders` collection
+   │        └── order.json
+```
+
+## Define information files
+The collection directory can contain multiple files with different extensions (json, js or ts). In this example consider using json files:
+
+**customer.json**
+```json
+[
+    {
+        "_id": { "$oid": "5e8508ea69a10e42d07414be" },
+        "name": "Emma",
+        "lastname": "Watson",
+        "email": "emma@watson.com",
+        "password": "$2a$10$hhn8vSvl6rpv42undbkQ.O5XIdIJkujyqQWqKIxRJgLQYCGlM/6rW"
+    },
+    {
+        "_id": { "$oid": "5e850bd669a10e42d07414c0" },
+        "name": "Pablo",
+        "lastname": "Picasso",
+        "email": "pablo@picasso.com",
+        "password": "$2a$10$hhn8vSvl6rpv42undbkQ.O5XIdIJkujyqQWqKIxRJgLQYCGlM/6rW"
+    }
+]
+```
+
+**menu.json**
+```json
+[
+    {
+        "_id": { "$oid": "5e6066bea0403411bc4a3333" },
+        "name": "Ceviche",
+        "price": 18.0,
+        "category": "firstCourse"
+    },
+    {
+        "_id": { "$oid": "5e6066e9a0403411bc4a3334" },
+        "name": "Tacos",
+        "price": 18.0,
+        "category": "firstCourse"
+    },
+    {
+        "_id": { "$oid": "5e606408dc8aec3040688a86" },
+        "name": "Lomo saltado",
+        "price": 20.0,
+        "category": "mainCourse"
+    },
+    {
+        "_id": { "$oid": "5e60650ea0403411bc4a332f" },
+        "name": "Stew",
+        "price": 20.0,
+        "category": "mainCourse"
+    }
+]
+```
+
+**order.json**
+
+```json
+[
+    {
+        "_id": { "$oid": "5e8186aa5d750a38e4260fcc" },
+        "order_detail": [
+            { "$oid": "5e606408dc8aec3040688a86" },
+            { "$oid": "5e6066bea0403411bc4a3333" }
+        ],
+        "customer": { "$oid": "5e8508ea69a10e42d07414be" },
+        "date": { "$date": { "$numberLong": "1583902800000" } }
+    },
+    {
+        "_id": { "$oid": "5e8187d53eda1f3a20d887e1" },
+        "order_detail": [
+            { "$oid": "5e60650ea0403411bc4a332f" },
+            { "$oid": "5e6066e9a0403411bc4a3334" }
+        ],
+        "customer": { "$oid": "5e850bd669a10e42d07414c0" },
+        "date": { "$date": { "$numberLong": "1583902800000" } }
+    }
+]
+```
+
+## Use Mongo Seeding CLI
+To use it in more than one project, you must install it globally with this command:
+
+```bash
+npm install -g mongo-seeding-cli
+```
+
+You can specify custom settings with command line parameters. The following example imports data from `./mongoose-express/example/data` directory using MongoDB connection URI mongodb://127.0.0.1:27017/mydb with option to drop database before import:
+> **NOTE:** You can use the [following parameters](../../cli/README.md#command-line-parameters) while using `seed` tool
+
+```bash
+seed -u mongodb://127.0.0.1:27017/dbname --drop-database ./data
+```
+
+After run the command it does not give us a message but if you look at your database GUI you will notice that you already have your tables populated with the information :rocket:
```

Comment:
```suggestion
Once the database seed finishes, the database is populated with collections from `./data` directory. 
```

Reply:
BTW:
> After run the command it does not give us a message

Hmm, wait... it shouldn't be like that. Mongo Seeding CLI should output detailed info what's going on... Did you discover a bug? 🤔

Reply:
Yes you did. Thanks a lot! :smile:

https://github.com/pkosiec/mongo-seeding/issues/114